src/internal_rules.jl

```diff
@@ -621,20 +621,20 @@ end
 function EnzymeRules.forward(
         func::Const{typeof(ldiv!)},
         RT::Type,
-        fact::Annotation{C},
-        B,
+        fact::Annotation{<:Cholesky},
+        B::Union{Const{<:AbstractVecOrMat}, DuplicatedNoNeed{<:AbstractVecOrMat}, Duplicated{<:AbstractVecOrMat}, BatchDuplicatedNoNeed{<:AbstractVecOrMat}, BatchDuplicated{<:AbstractVecOrMat}};
         kwargs...
-) where {C <: Union{Cholesky,Array}}
+)
     if isa(B, Const)
         @assert (RT <: Const)
-        return func.val(fact.val, B.val; kwargs...)
+        return ldiv!(fact.val, B.val; kwargs...)
```

Comment:
I would keep this as func.val just in case ldiv is overridden somewhere and ends up not referring to the expected function.

Reply:
```suggestion
        return func.val(fact.val, B.val; kwargs...)
```